cli/cmd/testdata/install_output.golden

```diff
@@ -117,101 +117,164 @@ spec:
+        - destination
+        - -addr=:8089
+        - -metrics-addr=:9999
+        - -log-level=ControllerLogLevel
         image: ControllerImage
         imagePullPolicy: ImagePullPolicy
-        args:
-        - "destination"
-        - "-addr=:8089"
-        - "-metrics-addr=:9999"
-        - "-log-level=ControllerLogLevel"
-      - name: proxy-api
+        name: destination
         ports:
-        - name: grpc
-          containerPort: 8086
-        - name: admin-http
-          containerPort: 9996
+        - containerPort: 8089
+          name: grpc
+        - containerPort: 9999
+          name: admin-http
+        resources: {}
+      - args:
+        - proxy-api
+        - -addr=:8086
+        - -metrics-addr=:9996
+        - -destination-addr=:8089
+        - -telemetry-addr=:8087
+        - -log-level=ControllerLogLevel
         image: ControllerImage
         imagePullPolicy: ImagePullPolicy
-        args:
-        - "proxy-api"
-        - "-addr=:8086"
-        - "-metrics-addr=:9996"
-        - "-destination-addr=:8089"
-        - "-telemetry-addr=:8087"
-        - "-log-level=ControllerLogLevel"
-      - name: tap
+        name: proxy-api
         ports:
-        - name: grpc
-          containerPort: 8088
-        - name: admin-http
-          containerPort: 9998
+        - containerPort: 8086
+          name: grpc
+        - containerPort: 9996
+          name: admin-http
+        resources: {}
+      - args:
+        - tap
+        - -addr=:8088
+        - -metrics-addr=:9998
+        - -log-level=ControllerLogLevel
         image: ControllerImage
         imagePullPolicy: ImagePullPolicy
-        args:
-        - "tap"
-        - "-addr=:8088"
-        - "-metrics-addr=:9998"
-        - "-log-level=ControllerLogLevel"
-      - name: telemetry
+        name: tap
         ports:
-        - name: grpc
-          containerPort: 8087
-        - name: admin-http
-          containerPort: 9997
+        - containerPort: 8088
+          name: grpc
+        - containerPort: 9998
+          name: admin-http
+        resources: {}
+      - args:
+        - telemetry
+        - -addr=:8087
+        - -metrics-addr=:9997
+        - -ignore-namespaces=kube-system
+        - -prometheus-url=http://prometheus:9090
+        - -log-level=ControllerLogLevel
         image: ControllerImage
         imagePullPolicy: ImagePullPolicy
-        args:
-        - "telemetry"
-        - "-addr=:8087"
-        - "-metrics-addr=:9997"
-        - "-ignore-namespaces=kube-system"
-        - "-prometheus-url=http://prometheus:9090"
-        - "-log-level=ControllerLogLevel"
-
-### Web ###
+        name: telemetry
+        ports:
+        - containerPort: 8087
+          name: grpc
+        - containerPort: 9997
+          name: admin-http
+        resources: {}
+      - env:
+        - name: CONDUIT_PROXY_LOG
+          value: warn,conduit_proxy=info
+        - name: CONDUIT_PROXY_CONTROL_URL
+          value: tcp://proxy-api.conduit.svc.cluster.local:8086
```

Comment:
As we discussed, I think this should be localhost

Reply:
> As we discussed, I think this should be localhost

As we discussed in the meeting, it's better for that that to happen in a separate PR that gets merged immediately after this one, so that we can compare the behavior of the localhost vs. proxy-api.conduit.svc.cluster.local configuration and so that the change to the golden file for the `localhost` is isolated.